src/components/cesium_map/api/spatial.js

```diff
@@ -305,6 +317,9 @@ export class ISamplesSpatial {
       terrainProvider: worldTerrain,
       fullscreenElement: element
     });
+    // limit the map max height
+    this.viewer.scene.screenSpaceCameraController.maximumZoomDistance = 20000000;
+    this.viewer.scene.screenSpaceCameraController.minimumZoomDistance = 10;
```

Comment:
Set the map minimum and maximum heights.

Reply:
Please add an explanation of what these values represent.

Reply:
For items like this it can be helpful to add a link to the reference docs for a more detailed explanation. https://cesium.com/learn/cesiumjs/ref-doc/ScreenSpaceCameraController.html#maximumZoomDistance